### PR TITLE
feature/prime-gov-minimal

### PR DESCRIPTION
**Pull request by dphoria**
### Link to Relevant Issue

This pull request is part of #101 

### Description of Changes

Implements bare minimum scraping for [PrimeGov](https://www.primegov.com/), leveraging  [`civic_scraper.platforms.primegov.site.PrimeGovSite`](https://github.com/biglocalnews/civic-scraper).

```Python
from cdp_scrapers.prime_gov_utils import PrimeGovScraper
from datetime import datetime
s = PrimeGovScraper("lacity", "America/Los_Angeles")
print(s.get_events(datetime(2022, 9, 1), datetime(2022, 9, 1))[0])

EventIngestionModel(
    body=Body(name='Energy, Climate Change, Environmental Justice, and River Committee', is_active=True, start_datetime=None, description=None, end_datetime=None, external_source_id=None),
    sessions=[Session(session_datetime=datetime.datetime(2022, 9, 1, 10, 0), video_uri='https://youtube.com/watch?v=v1vUDEQ5Ylw', session_index=0, caption_uri=None, external_source_id=None)],
    event_minutes_items=None,
    agenda_uri=None,
    minutes_uri=None,
    static_thumbnail_uri=None,
    hover_thumbnail_uri=None,
    external_source_id='13313'
)
```

Still more work to do. This is initial check-in.

### CODE REVIEW

**Comment by dphoria** [comment]
@Shak2000 Let me know if you have any comments on this MR.

File: cdp_scrapers/prime_gov_utils.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+
+class PrimeGovScraper(PrimeGovSite, IngestionModelScraper):
+    """
+    Adapter for civic_scraper PrimeGovSite in cdp-scrapers
+
+    See Also
+    --------
+    civic_scraper.platforms.primegov.site.PrimeGoveSite
+    cdp_screapers.scraper_utils.IngestionModelScraper
```
Comment (dphoria) [inline cdp_scrapers/prime_gov_utils.py:92]: Should be `cdp_scrapers...` 😅

**Review by evamaxfield — APPROVED** [review]
This is gorgeous. I have no notes.

File: cdp_scrapers/prime_gov_utils.py (hunk 1 above)
Comment (evamaxfield) [reply cdp_scrapers/prime_gov_utils.py:92]: We can fix that docstring later haha

**Pull request merged**